translated new files in categories section (#32)
ZupIT/opensource-tech-radar: content/en/categories/tools/nodemon.md

type="post"
layout="details"
isNew="true"

> description="Nodemon is a tool that optimizes the developers' time, with it it is possible to automatically restart projects with base in node.js when changes in the files are detected."

Reviewer: ```suggestion
description="Nodemon is a tool that optimizes the developers' time. You can automatically restart projects based on node.js when changes in the files are detected."
```

+++

{{% tabs/group %}}

>   {{% tabs/item id="0" title="What is it?" content="<p>Nodemon is a tool that optimizes the developers' time, with it it is possible to automatically restart projects with base in node.js when changes in the files are detected.</p>" %}}

Reviewer: ```suggestion
  {{% tabs/item id="0" title="What is it?" content="<p>Nodemon is a tool that optimizes the developers' time. You can automatically restart projects based on node.js when changes in the files are detected.</p>" %}}
```

>   {{% tabs/item id="1" title="Why?" content="<p>It is used in Web projects to facilitate development and productivity.</p>" %}}

Reviewer: ```suggestion
  {{% tabs/item id="1" title="How we use it?" content="<p>It is used in Web projects to facilitate development and productivity.</p>" %}}
```
